README.md

```diff
@@ -1,52 +1,57 @@
 # River - Decentralized Chat on Freenet
 
-![Screenshot of chat interface](screenshot-20241009.png)
-
-## Current Status (Jan 2025)
-
-River is currently in active development. We're working towards an alpha release by end of January 2025. Key features currently implemented:
-
-- Basic chat functionality
-- Room creation and management
-- Invitation-based membership
-- Message signing and verification
-- Web interface foundation
-
-Please note this is pre-release software - expect breaking changes and missing features. We welcome testing and feedback from early adopters!
-
 River is a decentralized group chat system built on Freenet, designed to provide a secure and
 upgradeable alternative to traditional chat platforms. It features a web-based interface built
 with [Dioxus](https://dioxuslabs.com) and a modular contract architecture using the freenet-scaffold
 framework.
 
-## Project Structure
+![Screenshot of chat interface](screenshot-20241009.png)
 
-- [common](common/): Shared code for contracts and UI
-- [ui](ui/): Web-based user interface
-- [contracts](contracts/): River chat room contract implementation
+## Current Status (Jan 2025)
 
-## Key Features
+River is currently in active development. We're working towards an alpha release by end of January 2025. Key features currently implemented:
 
 🌐 **Web-based Interface** - Modern web UI built with Dioxus for cross-platform compatibility  
 🔒 **Secure by Design** - Uses elliptic curve cryptography for authentication and signing  
 🔄 **Upgradeable** - Flexible upgrade mechanism for both UI and contracts  
 🌱 **Extensible** - Open architecture allows alternative UIs and integrations  
 📜 **Modular Contracts** - Built using freenet-scaffold for composable state management  
 📦 **Efficient Storage** - Uses CBOR serialization via [ciborium](https://crates.io/crates/ciborium)  
+✉️ **Invitation-based membership** - Requiring invitations helps combat spam
+
+Please note this is pre-release software - expect breaking changes and missing features. We welcome testing and feedback from early adopters!
```

Comment:
Need to be clear that as of right now it's not usable, don't want people trying it only to be disappointed.

Reply:
I think I made that clearer now, let me know what you think!